net/databag/Makefile

```
@@ -0,0 +1,65 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=databag
+PKG_VERSION:=1.1.11
```

```suggestion
PKG_VERSION:=1.1.11
PKG_RELEASE:=1
```

missing `PKG_RELEASE`